token/sdk/tms/db/tms.go

```diff
@@ -96,6 +98,20 @@ func (p *PostInitializer) ConnectNetwork(networkID, channel, namespace string) e
 		); err != nil {
 			return errors.WithMessagef(err, "failed to add processor to orion network [%s]", tmsID)
 		}
+		ttxDB, err := ttxdb.GetByTMSId(p.sp, tmsID)
```

Comment:
This could be reused:
```
func newTransactionFilter(sp ServiceProvider tmsID TMSID) TransactionFilter {
ttxDB, err := ttxdb.GetByTMSId(sp, tmsID)
		if err != nil {
			return errors.WithMessagef(err, "failed to get transaction db for [%s]", tmsID)
		}
		auditDB, err := auditdb.GetByTMSId(p.sp, tmsID)
		if err != nil {
			return errors.WithMessagef(err, "failed to get audit db for [%s]", tmsID)
		}
		return &TransactionFilter{
			ttxDB:   ttxDB,
			auditDB: auditDB,
		}
}
```

Reply:
👍